src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -242,4 +289,32 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
   trackByFn(_index: any, item: any) {
     return item.created;
   }
+
+  updateSelection(selection: CdTableSelection) {
+    this.selection = selection;
+  }
+
+  daemonAction(actionType: string) {
+    if (this.selection.first()?.daemon_type !== 'mon' &&
+      this.selection.first()?.daemon_type !== 'mgr') {
+      this.daemonService.action(this.selection.first()?.daemon_name, actionType)
+        .pipe(
+          take(1)
+        ).subscribe({
+          next: (resp) => {
+            console.log(resp)
+            this.notificationService.show(NotificationType.success, 'Daemon action scheduled',
+                                          resp.body.toString());
+          },
+          error: (resp) => {
+            console.log(resp)
+            this.notificationService.show(NotificationType.error, 'Daemon action failed',
+                                          resp.body.toString());
+          }
+        });
+    } else {
+            this.notificationService.show(NotificationType.error, 'Daemon action not valid',
+                                          'Daemon actions on mgr and mon daemons are not allowed');
```

Comment:
this block can be removed if actions are disabled for `mon` and `mgr`.

Reply:
Maybe it could be useful leaving it there?